spec/Dynamic-Validator.md

```diff
@@ -209,6 +212,14 @@ The revoke occurs immediately, but the validator cannot be ousted before its ter
 
 The transaction fails when the delegator revokes more than it delegates.
 
+
+### REDELEGATE
+* previous_delegatee
+* next_delegatee
+* quantity
+
+This is an atomic version of `REVOKE (previous_delegatee, quantity)` + `DELEGATE (next_delegatee, quantity)`. It works like two transactions are applied in a sequence, but the effect is atomic. The restrictions of the transaction are the same with both `REVOKE` and `DELGATE`.
```

Comment:
It works as if two transactions are applied in a sequence, but the effect is atomic.